clang/include/clang/Driver/Options.td

```diff
@@ -9043,6 +9043,11 @@ def dxil_validator_version : Option<["/", "-"], "validator-version", KIND_SEPARA
   HelpText<"Override validator version for module. Format: <major.minor>;"
            "Default: DXIL.dll version or current internal version">,
   MarshallingInfoString<TargetOpts<"DxilValidatorVersion">, "\"1.8\"">;
+def res_may_alias : Option<["/", "-"], "res-may-alias", KIND_FLAG>,
+  Group<dxc_Group>, Flags<[HelpHidden]>,
+  Visibility<[DXCOption, ClangOption, CC1Option]>,
+  HelpText<"Assume that UAVs/SRVs may alias">,
```

Comment:
This HelpText [matches that found in DXC](https://github.com/microsoft/DirectXShaderCompiler/blob/ec5324d66215cd748c162e0e5efed9a85b402ff9/include/dxc/Support/HLSLOptions.td#L547).
But I am not sure if it is an accurate description. Doesn't the flag only affect UAVs?
Perhaps the mention of SRVs should be removed?